samples/rtl/reciprog_double_pump.py

```diff
@@ -0,0 +1,343 @@
+import dace
```

Comment:
Does this sample show anything different than `axpy`? Otherwise we could get rid of it?

Reply:
It has been removed in https://github.com/carljohnsen/dace/commit/11984d3c2c997860d2124ef82133077ccd84e753